### DISCUSSION TITLE
Show individual Aim, Speed, and Accuracy Stats

### DISCUSSION THREAD

**Discussion by Finadoggie** [oamaok/ezpp#116] [Ideas]
Would it be possible for you guys to add the ability to check a map's individual Aim, Speed, and Acc values?  I think it'd be a nice utility to have for anybody who needs those values for whatever reason, since this extension is significantly faster than checking each .osz file individually with oppai, and even though the numbers won't be perfectly accurate, they give a good estimate.

(and just to avoid confusion, I'm talking about the three values highlighted here)
![image](https://user-images.githubusercontent.com/75299710/115095414-10fae480-9ed6-11eb-8543-5a829839f45d.png)

**Comment by acrylic-style** [comment]
Related: #70, #71, #119 (new)

There is initial implementation above (which is already removed in HEAD), but we need design overhaul for this.

Answer: Yes, but only when we have a good design. Or maybe tooltips could be better.
